docs/releases.md

```diff
@@ -1,5 +1,70 @@
 # DietPi Releases
 
+## May 2021 (version 7.2 - Beta)
+
+### New optimised software in 7.2
```

Comment:
Btw, the following is possible when we don't want to have the version string repeated on every header:
```
### New optimised software {: id="new-optimised-software-7.2" }
```
So the visual header and the table of content entry won't have the version string, but the URL when clicking the link or the TOC entry will be `https://dietpi.com/docs/releases/#new-optimised-software-7.2` and hence be unique. That's nice for repeating headers in the software section as well.

Do we want to keep the "optimised" here now that it does not appear in the `dietpi-software` script?

Reply:
Indeed, I like a lot the way of setting the IDs, and I will apply the changes, both in 7.2 and 7.1. 
The `optimised` keywork is indeed not necessary.

Reply:
In the headers the in this case (much content) the version string is does not hurt much IMO, but the TOC will look much cleaner without it 😃.